pkg/jsonapi/resource.go

```diff
@@ -240,6 +240,30 @@ func (r *Resource) SaveAsMultipart(fields []string) error {
 	return nil
 }
 
+/*
+Delete a resource from the server. Response is empty on success
+*/
+func (r *Resource) Delete() error {
```

Comment:
Optional: the Python SDK, after getting an ok response from the server, sets the resource's ID to `None`. The idea is that you should be able to:

```python
team.delete()
team.save('organization', 'name')
```

which would create a **new** team with the same attributes and relationships as the old.

https://github.com/transifex/transifex-python/blob/devel/transifex/api/README.md#deleting